scripts/travis.sh

```diff
@@ -8,7 +8,7 @@ IOHK_OPS=${1:-$(nix-build -A iohk-ops)/bin/iohk-ops}
 CLEANUP_DEPLOYS=${2:-true}
 CLEANUP_CONFIGS=${3:-true}
 WITH_STAGING=${4:-true}
-WITH_PRODUCTION=${5:-true}
+WITH_PRODUCTION=${5:-}
```

Comment:
Later we branch on whether `$WITH_PRODUCTION` is an empty string or not. Is the only way to pass arguments 6, 7, and 8 while keeping `$WITH_PRODUCTION` empty using a `""` placeholder? Is that what we want? I see `.travis.yml` doesn't pass any arguments, so it's using the default values, in this case an empty string.

Reply:
It is, correct.

Reply:
And my taste tells me it's better, when it's parametrisable : -)

..esp when you do local testing iterations, where you want to eval different parts.

Reply:
Works for me. Thanks for clarifying.

Reply:
This alone won't be enough, best to skip CI for now and revert this change.

Reply:
Done.